src/datachain/toolkit/split.py

```diff
@@ -56,16 +67,28 @@ def train_test_split(dc: DataChain, weights: list[float]) -> list[DataChain]:
     if any(weight < 0 for weight in weights):
         raise ValueError("Weights should be non-negative")
 
-    weights_normalized = [weight / sum(weights) for weight in weights]
+    def scale(multiplier: float) -> int:
+        if multiplier < 0:
+            multiplier = 0.0
+        if multiplier > 1:
+            multiplier = 1.0
```

Comment:
```suggestion
        multiplier = min(max(0, multiplier),1.0)
```

Reply:
Oh, awesome, thank you! 🙏 Always forget about this trick 😅